.appveyor.yml

```diff
@@ -1,21 +1,18 @@
 environment:
-  matrix:
-    - TOXENV: py27-cov
-    - TOXENV: py36-cov
+  global:
+    CIBW_SKIP: cp33-* cp34-* cp35-*
+    CIBW_TEST_REQUIRES: pytest pytest-cov
+    CIBW_TEST_COMMAND: "cd {project}\\tests && pytest --cov"
 
-install:
-  - git submodule update --init
-  - python -m pip install --upgrade pip
-  - pip install -r requirements.txt
-  - pip install -r dev-requirements.txt
+build_script:
+  - pip install cibuildwheel==0.9.2 coverage codecov
+  - cibuildwheel --output-dir wheelhouse
+  - coverage combine tests
+  - codecov
```

Comment:
@madig I think you can keep calling codecov via tox, as it was before `tox -e codecov`